Face_Recognition/Recognition.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import cv2 as cv
+import os
+haar_cascade = cv.CascadeClassifier('haar_face.xml')
+
+people = []
+DIR = r'images'  # DIR - path of the folder containing images
+for i in os.listdir(DIR):
+    people.append(i)
+
+# LBPHFaceRecognizer - instatiate the face recognizer
+face_recognizer = cv.face.LBPHFaceRecognizer_create()
+# We are using the trained yml file.
+face_recognizer.read('face_trained.yml')
+
+# img = cv.imread(r'images\henry\13.jpg')
```

Comment:
```suggestion
img = cv.imread(r'images\henry\13.jpg')
```

Reply:
I have tried this, but the same error is occurring.